genie-web/src/main/resources/application.yml

```diff
@@ -51,6 +51,7 @@ genie:
     users:
       creationEnabled: false
       runAsUserEnabled: false
+      concurrentJobLimit: 0
```

Comment:
Bug. Did a last minute rename in the code and missed this one. Fixed now (and became 2 variables, enable and value).